Makefile

```
@@ -65,7 +65,46 @@ python_sdk::
 		rm ./bin/setup.py.bak && \
 		cd ./bin && python3 setup.py build sdist
 
+gen_examples::
+	cd ${WORKING_DIR}/examples/yaml && \
+	rm -rf ${WORKING_DIR}/examples/{go,nodejs,python,dotnet} && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language go --out ${WORKING_DIR}/examples/go 2>&1 && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language nodejs --out ${WORKING_DIR}/examples/nodejs 2>&1 && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language python --out ${WORKING_DIR}/examples/python 2>&1 && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language dotnet --out ${WORKING_DIR}/examples/dotnet 2>&1
+
+pulumi_login::
+	export PULUMI_CONFIG_PASSPHRASE="asdfqwerty1234" && \
+	pulumi login --local
+
+pulumi_up::
+	export PULUMI_CONFIG_PASSPHRASE="asdfqwerty1234" && \
+	cd ${WORKING_DIR}/examples/yaml && \
+	pulumi stack init dev && \
+	pulumi stack select dev && \
+	pulumi config set name dev && \
+	pulumi up -y
+
+pulumi_down::
+	export PULUMI_CONFIG_PASSPHRASE="asdfqwerty1234" && \
+	cd ${WORKING_DIR}/examples/yaml && \
+	pulumi stack select dev && \
+	pulumi destroy -y && \
+	pulumi stack rm dev -y
+
+devcontainer::
+	git submodule update --init --recursive .devcontainer && \
+	git submodule update --remote --merge .devcontainer && \
+	cp -f .devcontainer/devcontainer.json .devcontainer.json
```

These don't need to be part of the same process.
```suggestion
	git submodule update --init --recursive .devcontainer
	git submodule update --remote --merge .devcontainer
	cp -f .devcontainer/devcontainer.json .devcontainer.json
```